### PR TITLE
Feature/scenarios

### PR DESCRIPTION
**Pull request by sharovik**

### CODE REVIEW

File: events/examplescenario/event.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	helpMessage = "Ask me `write a message` and you will see the answer."
+
+	//The migrations folder, which can be used for event installation or for event update
+	migrationDirectoryPath = "./events/examplescenario/migrations"
+
+	regexChannel = `(?im)(?:<#)(\w+)(?:\|)`
+)
+
+//ExmplEvent the struct for the event object. It will be used for initialisation of the event in defined-events.go file.
+type ExmplSnrEvent struct {
```
Comment (shoaibi) [inline events/examplescenario/event.go:31]: Is it a common practice in go to shorten names?

Comment (sharovik) [reply events/examplescenario/event.go:31]: Well
It's not a common practice, I will rename it to `EventStruct` name. Which will be shorter and more correct I think 🌝

**Comment by sharovik** [comment]
@shoaibi hey) can you please give me your feedback?)

File: internal/database/sqlite_test.go [hunk 2]
```diff
@@ -284,6 +284,46 @@ func TestSQLiteDictionary_InsertQuestion(t *testing.T) {
...
+
+	err := dictionary.InitSQLiteDatabaseConnection()
+	assert.NoError(t, err)
+	upTestTables(t)
+	insertTestData(t)
+
+	var questionID int64
+	questionID, err = dictionary.InsertQuestion(
+		"Hello bot",
+		"Yo",
```
Comment (shoaibi) [inline internal/database/sqlite_test.go:300]: YoLo ;)

**Review by shoaibi — APPROVED** [review]

**Pull request merged**